### PR TITLE
Fix #234 : Convert all ViewModels to ObservableViewModel 

### LINKED ISSUE #234: Convert all ViewModels to ObservableViewModel
There is inconsistency in terms of usage of ViewModel and ObservableViewModel only if there is something to observe in that file, so to bring consistency to code convert all ViewModels to ObservableViewModels.

### PR DESCRIPTION
**Pull request by prayutsu**
## Explanation

Fixes #234 
Converted all the ViewModels to ObservableViewModels

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by prayutsu** [comment]
@rt4914 PTAL

File: app/src/main/java/org/oppia/app/testing/BindableAdapterTestViewModel.kt [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 import androidx.lifecycle.MutableLiveData
-import androidx.lifecycle.ViewModel
 import org.oppia.app.fragment.FragmentScope
 import org.oppia.app.model.TestModel
+import org.oppia.app.viewmodel.ObservableViewModel
 import javax.inject.Inject
 
 // TODO(#59): Make this view model only included in relevant tests instead of all prod builds.
 /** A [ViewModel] for testing the bindable RecyclerView adapter. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/testing/BindableAdapterTestViewModel.kt]: Change this comment too.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Great. Thanks for the PR.
One nit change, in all the files that you have changed, please check the comment/documentation and update `ViewModel` word with `ObservableViewModel`

File: app/src/main/java/org/oppia/app/testing/BindableAdapterTestViewModel.kt (hunk 1 above)
Comment (prayutsu) [reply app/src/main/java/org/oppia/app/testing/BindableAdapterTestViewModel.kt]: done

Comment (prayutsu) [reply app/src/main/java/org/oppia/app/testing/BindableAdapterTestViewModel.kt]: Do I need to also update the comment in the files that I didn't change?

Comment (rt4914) [reply app/src/main/java/org/oppia/app/testing/BindableAdapterTestViewModel.kt]: No that should be fine. Just fix it in all the files that you have changed.

**Comment by prayutsu** [comment]
@rt4914 PTAL, only ```ViewModel``` has been replaced with ```ObservableViewModel```, neither ```view model``` or ```view Model```

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**